### PR TITLE
Adds Sensor section in the Azure providers docs 

### PR DESCRIPTION
**Pull request by Adaverse**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: airflow/providers/microsoft/azure/example_dag/__init__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (pragnareddye) [inline airflow/providers/microsoft/azure/example_dag/__init__.py:1]: Is there a related issue? or open epic? Just to get some context

Comment (Adaverse) [reply airflow/providers/microsoft/azure/example_dag/__init__.py:1]: It's not related to any issue or epic. There is a general agreement that if there is something (bug or feature) that you have decided to work on, then no need to open an issue for it. Just directly open the PR.

Comment (pragnareddye) [reply airflow/providers/microsoft/azure/example_dag/__init__.py:1]: Thank you!

**Comment by Adaverse** [comment]
cc - @potiuk

**Review by potiuk — APPROVED** [review]

**Pull request merged**